### PR TITLE
Remove ai sdk dependency.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Send the ai Data from rest API.

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java [hunk 1]
```diff
@@ -5,43 +5,56 @@
...
 
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON;
+
 public class TelemetrySender {
     private static final Logger log = LoggerFactory.getLogger(TelemetrySender.class);
 
     private static final int INSTRUMENTATION_KEY_LENGTH = 36;
 
-    private final TelemetryClient client;
+    private static final String TELEMETRY_TARGET_URL = "https://dc.services.visualstudio.com/v2/track";
```
Comment (warrenzhu25) [inline spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java]: Could you move all rest client logic into a class such as TelemetryRestClient, then just replace original client into this one? In the future, we just need to replace this client.

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java [hunk 2]
```diff
@@ -51,8 +64,36 @@ private static boolean isValid(String instrumentationKey) {
...
+
+        for (int i = 0; i < RETRY_LIMIT; i++) {
+            response = executeRequest(eventData);
+
+            if (response != null && response.getStatusCode() == HttpStatus.OK) {
+                return;
+            }
+        }
+
+        if (response != null && response.getStatusCode() != HttpStatus.OK) {
```
Comment (warrenzhu25) [inline spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java]: This if condition is not needed here.

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java [hunk 3]
```diff
@@ -51,8 +64,36 @@ private static boolean isValid(String instrumentationKey) {
...
+    private static ResponseEntity<String> executeRequest(final TelemetryEventData eventData) {
+        try {
+            final HttpEntity<String> body = new HttpEntity<>(MAPPER.writeValueAsString(eventData), HEADERS);
+
+            return REST_TEMPLATE.exchange(TELEMETRY_TARGET_URL, HttpMethod.POST, body, String.class);
+        } catch (Exception ignore) {
+            log.warn("Failed to exchange telemetry request, {}.", ignore.getMessage());
+        }
+
+        return null;
```
Comment (warrenzhu25) [inline spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java]: Suggest not returning null here.

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java (hunk 2 above)
Comment (Incarnation-p-lee) [reply spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java]: OK. Will refine to true or false.

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetrySender.java]: I use the same name with TelemetrySender for that purpose in other repo, but you have this name already. Will try new name for this.

**Pull request merged**